recipes/gpi_core/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2.0.0rc" %}
+{% set build_num = "0" %}
+{% set sha256 = "8dc97c93e48f57febc67773107276dbbdadb8c8effa75bea18a475c07266442a" %}
+
+package:
+  name: gpi_core
+  version: {{ version }}
+
+source:
+  url: https://github.com/gpilab/core-nodes/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  folder: repo_contents
+
+build:
+  number: {{ build_num }}
+  detect_binary_files_with_prefix: True
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  
+    - {{ compiler('cxx') }}
+  host:
+    - cytoolz
+    - eigen =3
+    - fftw
+    - gpi >=1.1.4
+    - numpy >=1.11
+    - python
+  run:
+    - {{ pin_compatible('eigen') }}
+    - {{ pin_compatible('fftw') }}
+    - {{ pin_compatible('gpi') }}
+    - matplotlib >=2.1,<3
+    - {{ pin_compatible('numpy') }}
+    - {{ pin_compatible('python') }}
```

Comment:
Replace with `- python`

Reply:
Will do, but if you don't mind explaining, why one and not the other? I'm not sure I understand the difference but I'm guessing it has to do with the global config settings on conda-forge?

Reply:
Correct setting is `{{ pin_compatible('python', min_pin="x.x", max_pin="x.x") }}` which is hard to remember and just `- python` does the same thing.

Reply:
Thanks!